### ISSUE TITLE
Not resolved: Faulty DNS records #1256

### ISSUE THREAD

**Issue by valuta1995** [linode/docs#1267]
Issue (#1256) not resolved. The mail.example.com should also be an A record. Note that MX records cannot refer to IPs [https://github.com/linode/docs/blob/master/docs/email/postfix/email-with-postfix-dovecot-and-mysql.md](https://github.com/linode/docs/blob/master/docs/email/postfix/email-with-postfix-dovecot-and-mysql.md)
```
example.com A 10 12.34.56.78
example.com MX 10 example.com
mail.example.com MX 10 12.34.56.78
```

**Comment by GuessWhoSamFoo** [comment]
@Guaris

**Comment by GuessWhoSamFoo** [comment]
Closed in favor of #1256
